release/golden_notebook_tests/golden_notebook_tests.yaml

```diff
@@ -7,7 +7,7 @@
   run:
     use_connect: True
     autosuspend_mins: 10
-    timeout: 1200
+    timeout: 1500
```

Comment:
Is this file still needed if this is in `release_tests.yaml`?

Reply:
Good point, I'll remove it - thanks!